examples/community.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+  <!-- build:css ../css/bootstrap.min.css -->
+  <link href="../html/css/bootstrap.min.css" rel="stylesheet">
+  <!-- /build -->
+  <!-- build:css ../css/style.css -->
+  <link href="../html/css/style.css" rel="stylesheet">
+  <!-- /build -->
+  <link href="common/css/style.css" rel="stylesheet">
+</head>
+<body>
+<div class="container">
+  <div class="page-header">
+    <h1>Community Examples</h1>
+    <p class="lead">
+    Here are a couple examples of what people are building with Rocky.js. Open a pull request to add yours!
+    </p>
+  </div>
+  <div class="community-example">
+    <h3>Simply Square by <a class="example-author" href="https://github.com/NiVZ78">NiVZ78</a></h3>
+    <p><a class="jsbin-embed" href="http://jsbin.com/wakuha/embed?js,output&height=350px">
+      Simply Square by NiVZ78 on jsbin.com
+    </a></p>
+    <script src="https://static.jsbin.com/js/embed.min.js?3.35.9"></script>
+  <div>
+</div>
```

Comment:
Ok will do

On Tuesday, January 26, 2016, Heiko Behrens <notifications@github.com>
wrote:

> In examples/community.html
> <https://github.com/pebble/rockyjs/pull/37#discussion_r50938936>:
>
> > +<body>
> > +<div class="container">
> > +  <div class="page-header">
> > +    <h1>Community Examples</h1>
> > +    <p class="lead">
> > +    Here are a couple examples of what people are building with Rocky.js. Open a pull request to add yours!
> > +    </p>
> > +  </div>
> > +  <div class="community-example">
> > +    <h3>Simply Square by <a class="example-author" href="https://github.com/NiVZ78">NiVZ78</a></h3>
> > +    <p><a class="jsbin-embed" href="http://jsbin.com/wakuha/embed?js,output&height=350px">
> > +      Simply Square by NiVZ78 on jsbin.com
> > +    </a></p>
> > +    <script src="https://static.jsbin.com/js/embed.min.js?3.35.9"></script>
> > +  <div>
> > +</div>
>
> if you add the
>
> <!-- build:template
> <%= github_banner %>
> /build -->
>
> from the other pages it might encourage even more to fork / create PRs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pebble/rockyjs/pull/37/files#r50938936>.
>